docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:20.04
```

Comment:
can you move this file in the root directory itself?

Reply:
I think it doesn't have any issue with anything, I keep it separate to make things organized.